statsmodels/regression/linear_model.py

```diff
@@ -57,6 +57,8 @@
 from statsmodels.tools.typing import Float64Array
 from statsmodels.tools.validation import bool_like, float_like, string_like
 
+from wildboottest.wildboottest import wildboottest
```

Comment:
what is wildboottest?
is this a package or module?
can we include it in statsmodels.

Reply:
`wildboottest` is a package that implements the wild cluster bootstrap.

https://github.com/s3alfisc/wildboottest

I'll let @s3alfisc comment as well, but I think we'd like to keep the package separate, as there are more features to add, and we want to ultimately make a PR for `linearmodels` as well.